modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -267,110 +268,233 @@
 			<return type="void" />
 			<description>
+				Export a [String] property as a path to a directory. The path will be limited to the project folder and its subfolders. See [annotation @export_global_dir] to allow picking from the entire filesystem.
+				See also [constant PROPERTY_HINT_DIR].
+				[codeblock]
+				@export_dir var sprite_folder: String
+				[/codeblock]
 			</description>
 		</annotation>
 		<annotation name="@export_enum" qualifiers="vararg">
 			<return type="void" />
 			<param index="0" name="names" type="String" />
 			<description>
+				Export a [String] or integer property as an enumerated list of options. If the property is an integer field, then the index of the value is stored, in the same order the values are provided. You can add specific identifiers for allowed values using a colon.
+				See also [constant PROPERTY_HINT_ENUM].
+				[codeblock]
+				@export_enum("Rebecca", "Mary", "Leah") var character_name: String
+				@export_enum("Warrior", "Magician", "Thief") var character_class: int
+				@export_enum("Elf:1", "Dwarf:3", "Orc:8") var character_race: int
+				[/codeblock]
 			</description>
 		</annotation>
 		<annotation name="@export_exp_easing" qualifiers="vararg">
 			<return type="void" />
 			<param index="0" name="hints" type="String" default="&quot;&quot;" />
 			<description>
+				Export a floating-point property with an easing editor widget. Additional hints can be provided to adjust the behavior of the widget. [code]"attenuation"[/code] flips the curve, which makes it more intuitive for editing attenuation properties. [code]"positive_only"[/code] limits values to only be greater than or equal to zero.
+				See also [constant PROPERTY_HINT_EXP_EASING].
+				[codeblock]
+				@export_exp_easing var transition_speed
+				@export_exp_easing("attenuation") var fading_attenuation
+				@export_exp_easing("positive_only") var effect_power
+				[/codeblock]
 			</description>
 		</annotation>
 		<annotation name="@export_file" qualifiers="vararg">
 			<return type="void" />
 			<param index="0" name="filter" type="String" default="&quot;&quot;" />
 			<description>
+				Export a [String] property as a path to a file. The path will be limited to the project folder and its subfolders. See [annotation @export_global_file] to allow picking from the entire filesystem.
+				If [param filter] is provided, only matching files will be available for picking.
+				See also [constant PROPERTY_HINT_FILE].
+				[codeblock]
+				@export_file var sound_effect_file: String
+				@export_file("*.txt") var notes_file: String
+				[/codeblock]
 			</description>
 		</annotation>
 		<annotation name="@export_flags" qualifiers="vararg">
 			<return type="void" />
 			<param index="0" name="names" type="String" />
 			<description>
+				Export an integer property as a bit flag field. This allows to store several "checked" or [code]true[/code] values with one property, and comfortably select them from the Inspector dock.
```

Comment:
```suggestion
				Export an integer property as a bit flag field. This allows to store several boolean "checked" or [code]true[/code]/[code]false[/code] values with one property, and comfortably select them from the Inspector dock.
```

Reply:
I'm not sure about that. It's phrased like it is to try and explain the bit flag field in both layman's terms and in technical terms. Adding "boolean" IMO negates the layman's bit. And adding "false" is probably unnecessary, because false values aren't really stored in any way, technically. So I've left this untouched thus far.